pacsman/dicom_interface.py

```diff
@@ -110,8 +110,8 @@ def fetch_images_as_files(self, series_id):
     @abstractmethod
     def fetch_thumbnail(self, series_id):
         """
-        Fetches a central slice of a series from PACS
+        Fetches a central slice of a series from PACS and converts to PNG
         :param series_id: SeriesInstanceUID from PACS
-        :return: A path to a dicom file on success, None if not found
+        :return: A path to a PNG file on success, None if not found or failure
```

Comment:
Why return `None` on failure, as opposed to allowing the exception to bubble up?

Reply:
This is here because`dicom-numpy` throws an exception on most of the test images (the ones that came from www.dicomstandard.co.uk) because they lack an `ImageOrientationPatient` attribute. 

One could also make a case that thumbnails aren't critical enough to throw a user facing exception for, instead of just showing a blank space.

Reply:
I see; it is interesting that most of the test images don't have the `ImageOrientationPatient` tag.

This brings up another question: why do we need dicom-numpy at all for the thumbnails?  I know we talked about needing it for some reason, but I think I must have bee confused---It doesn't seem like we need it (yet) for pacsman.  We will likely need it later though.

See my other comment.

Regarding whether thumbnails are important or not: whether or not the thumbnails are important seems like it could vary a lot from application to application.

Reply:
Without dicom-numpy around, the exception doesn't get thrown, so I removed the `except` blocks.

Reply:
Awesome! Do we need to update the doctoring?